src/WordPress/WordPressClientAPI.php

```diff
@@ -15,10 +15,10 @@ class WordPressClientAPI extends Client
     public function getZoneTag($zone_name)
     {
         $request = new Request('GET', 'zones/', array('name' => $zone_name), array());
-        $response = self::callAPI($request);
+        $response = $this->callAPI($request);
```

Comment:
Oh my bad, this is what I get for doing code reviews late at night.  Yeah you changed it correctly.